** M-x report-emacs-bug includes the package version.
   Provided by: Romain Francoise <rfrancoise@debian.org>

--- emacs-snapshot-20070902.orig/lisp/mail/emacsbug.el
+++ emacs-snapshot-20070902/lisp/mail/emacsbug.el
@@ -146,6 +146,11 @@
     (add-text-properties (1+ user-point) (point) prompt-properties)
 
     (insert "\n\nIn " (emacs-version) "\n")
+    (insert " (emacs-snapshot package, version "
+	    (substring (shell-command-to-string
+			"dpkg -s emacs-snapshot-common | grep ^Version:")
+		       9 -1)
+	    ")\n")
     (if (fboundp 'x-server-vendor)
 	(condition-case nil
             ;; This is used not only for X11 but also W32 and others.
